qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -187,7 +195,7 @@ def write_ceph_exclude(self):
         # These tests will fail or take too much time and will
         # make the test timedout, just skip them for now.
         xfstests_exclude_contents = dedent('''\
-            {c}/001 {g}/003 {g}/020 {g}/075 {g}/317 {g}/538 {g}/531
```

Comment:
Technically, it would better to have a separate commit for this change since it doesn't fit with rest of the changes in this commit.

Reply:
Done and fixed them all.